pkg/serverless/trace/install.json

```diff
@@ -0,0 +1 @@
+{"install_id":"000001f5-952a-21ee-9801-0e0c566f5cb7","install_type":"manual","install_time":1701972019}
```

Comment:
These `install.json` files should not be committed.

Reply:
This is un-intended. Removed the file.